manager/integration/tests/common.py

```diff
@@ -4467,6 +4475,32 @@ def wait_for_pod_annotation(core_api,
     assert matches is True
 
 
+def wait_for_volume_clone_status(client, name, key, value):
+    for _ in range(RETRY_COUNTS):
+        volume = client.by_id_volume(name)
+        if volume[VOLUME_FIELD_CLONE_STATUS][key] == value:
+            break
+        time.sleep(RETRY_INTERVAL)
+    assert volume[VOLUME_FIELD_CLONE_STATUS][key] == value, \
+        f" Expected value={value}\n. " \
+        f" Got volume[{VOLUME_FIELD_CLONE_STATUS}][{key}]= " \
+        f"{volume[VOLUME_FIELD_CLONE_STATUS][key]}\n. volume={volume}"
+    return volume
+
+
+def get_clone_volume_name(client, source_volume_name):
+    for _ in range(RETRY_EXEC_COUNTS):
+        volumes = client.list_volume()
+        for volume in volumes:
+            if volume['cloneStatus']['sourceVolume'] == \
+                    source_volume_name:
+                return volume.name
+            else:
+                continue
```

Comment:
nit: we don't need this else condition, because at the end of loop, it will just reiterate the next.

Reply:
Right, we don't need it, removed it. :)